app/controllers/admin_controller.rb

```diff
@@ -102,4 +104,29 @@ def culled_games
   def about
   end
 
+  def stats_display
+    games = Event.current.top_games()
+    titles = Game.where(culled: false).size
+    active_checkouts = Checkout.where(event: Event.current, closed: false).size
+    checked_out = number_to_percentage(active_checkouts.to_f / titles, precision: 0)
+    titles_played = Checkout.where(event: Event.current).joins(game: [:title]).group(:title).count.count
+    total_checkouts = Checkout.where(event: Event.current).size
+    longest_checkout = Checkout.longest_checkout_game_today(Event.current.utc_offset)
```

Comment:
Passing in the offset here is weird and not obvious why it's needed. Why can't you do it in the class method, since the current event is globally accessible?

Reply:
Might not be for the current event?